torch/_dynamo/bytecode_transformation.py

```diff
@@ -289,6 +312,81 @@ def _update(delta, size):
     return linetable, update
 
 
+@dataclasses.dataclass
+class ExceptionTableEntry:
+    start: int
+    end: int
+    target: int
+    depth: int
+    lasti: bool
+
+
+def encode_exn_tab_varint(n):
+    # unfortunately cannot use encode_varint since the order of
+    # the bytes is reversed for exception tables.
```

Comment:
Add docstring explaining that this encoding is, use full words for function names.